### DISCUSSION TITLE
SAP2000&IDEA Statica

### DISCUSSION THREAD

**Discussion by zjx252101782** [idea-statica/ideastatica-public#333] [Features and Ideas]
Can it be developed to directly import SAP2000 model files, such as SDB or S2K

**Comment by nathanlukeIDEA** [comment]
Hi @zjx252101782. Please see our page on SAP2000 link. https://www.ideastatica.com/bim/sap2000steel. The link works directly with the SAP2000 API. We do not currently provide a way of importing a SDB or S2K directly into Checkbot. 

Can you please elaborate on why this feature would be beneficial for you?
